apis/v1beta1/types.go

```diff
@@ -203,6 +203,21 @@ type VirtualMachineCloneSpec struct {
 	// Check the compatibility with the ESXi version before setting the value.
 	// +optional
 	HardwareVersion string `json:"hardwareVersion,omitempty"`
+	// DataDisks are additional disks to add to the VM that are not part of the VM's OVA template.
+	// +optional
```

Comment:
```suggestion
	// +optional
	// +listType=map
	// +listMapKey=name
```

Reply:
I was guessing 29 being a good fit as that's the maximum we currently handle in the code?

Note: we can start lower and increase later, but cannot decrease without be api breaking.

Reply:
good point.

> [...] Order is preserved upon merge. [...]

https://github.com/kubernetes/kube-openapi/blob/master/pkg/idl/doc.go#L48-L51